src/gui/gui.cpp

```diff
@@ -62,10 +88,15 @@ void init()
 	for(const config& g : guis_cfg.child_range("gui")) {
 		const std::string id = g["id"];
 
-		auto iter = guis.emplace(id, gui_definition(g)).first;
+		auto [iter, is_unique] = guis.emplace(id, gui_definition(g));
 
-		if(id == "default") {
-			default_gui = iter;
+		if (!is_unique) {
+			ERR_GUI_P << "GUI Theme ID '" << id << "' already exists.";
+			current_gui = default_gui;
+		} else {
+			if(id == "default") {
```

Comment:
I'll need to check if the three lines here make any sense either. A duplicate "default" won't be inserted anyway.

Reply:
I think it's correct though IMO even better would be to only check mainline themes for this.

Reply:
> IMO even better would be to only check mainline themes for this.

Done.